common/common-lang/src/test/java/com/twelvemonkeys/util/CollectionAbstractTest.java

```diff
@@ -1136,7 +1136,7 @@ public void testCollectionToArray2() {
         verifyAll();
 
         try {
-            array = collection.toArray(null);
+            array = collection.toArray((Object[])null);
```

Comment:
This change looks ok. Please insert a space between the cast and the `null` to follow code style.

Reply:
Okay. I added the space. I can squash the commits eventually if you'd prefer. Just let me know once I get it right. Thanks.